front/app/containers/MainHeader/DesktopNavbar/getNavbarItemPropsArray.ts

```diff
@@ -0,0 +1,22 @@
+import { INavbarItem, getNavbarItemSlug } from 'services/navbar';
```

Comment:
The disadvantage of this way of getting props is that you don't benefit from TS functionalities to e.g. see where/if a prop is used.

Reply:
Hmm would that be solved by instead of doing

```
<FullMobileNavMenuItem {...navbarItemProps}
```

Doing something like

````
<FullMobileNavMenuItem linkTo={navbarItemProps.linkTo} onlyActiveOnIndex={navbarItemProps.onlyActiveOnIndex}
```

?

Reply:
Yes, that would solve it!